src/booleans.js

```diff
@@ -1,57 +1,57 @@
 const negate = (a) => {
-  // your code here
+  return !a;
 };
 
 const both = (a, b) => {
-  // your code here
+  return (a && b);
 };
 
 const either = (a, b) => {
-  // your code here
+  return (a || b);
 };
 
 const none = (a, b) => {
-  // your code here
+return !(a || b); 
 };
 
 const one = (a, b) => {
-  // your code here
+  return !(a == b);
 };
 
 const truthiness = (a) => {
-  // your code here
+  return !!(a);
 };
 
 const isEqual = (a, b) => {
-  // your code here
+  return (a == b);
 };
 
 const isGreaterThan = (a, b) => {
-  // your code here
+  return a > b;
 };
 
 const isLessThanOrEqualTo = (a, b) => {
-  // your code here
+  return a <= b;
 };
 
 const isOdd = (a) => {
-  // your code here
+  return a % 2 == 1;
 };
 
-const isEven = (a) => {
-  // your code here
+  const isEven = (a) => {
+  return a % 2 == 0;
 };
 
 const isSquare = (a) => {
-  // your code here
+  return Math.sqrt(a) % 1 === 0;
 };
 
 const startsWith = (char, string) => {
-  // your code here
+  return string.startsWith('');
 };
 
 const containsVowels = (string) => {
-  // your code here
+  return 'aeiou'.indexOf( chr[0].toLowerCase() ) !== -1;
```

Comment:
When you pass one argument to a method (or a function), it shouldn't have any whitespace before or after. When you pass multiple arguments, you should separate arguments with a comma and a space (but not the last argument). E.g 

```js
'aeiou'.indexOf(chr[0].toLowerCase()) !== -1;
```

```js
console.log('argument1', 'argument2');
```